pallets/claims/src/lib.rs

```diff
@@ -94,14 +94,16 @@ impl StatementKind {
 	/// Convert this to the (English) statement it represents.
 	fn to_text(self) -> &'static [u8] {
 		match self {
-			StatementKind::Regular =>
+			StatementKind::Regular => {
 				&b"I hereby agree to the terms of the statement whose sha2256sum is \
 				5627de05cfe235cd4ffa0d6375c8a5278b89cc9b9e75622fa2039f4d1b43dadf. (This may be found at the URL: \
-				https://statement.tangle.tools/airdrop-statement.html)"[..],
-			StatementKind::Safe =>
+				https://statement.tangle.tools/airdrop-statement.html)"[..]
+			},
+			StatementKind::Safe => {
 				&b"I hereby agree to the terms of the statement whose sha2256sum is \
 				7eae145b00c1912c8b01674df5df4ad9abcf6d18ea3f33d27eb6897a762f4273. (This may be found at the URL: \
-				https://https://statement.tangle.tools/safe-claim-statement)"[..],
+				https://https://statement.tangle.tools/safe-claim-statement)"[..]
```

Comment:
extra `https` is needed?

Reply:
I don't think so, removing it. Seems like it was added by mistake